test/testenv/cmutil.go

```diff
@@ -0,0 +1,60 @@
+package testenv
+
+import (
+	"encoding/json"
+	"fmt"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+// ClusterMasterHealthResponse is a representation of the health response by a Splunk cluster-master
+// Endpoint: /services/cluster/master/health
+type ClusterMasterHealthResponse struct {
+	Entries []ClusterMasterHealthEntry `json:"entry"`
+}
+
+// ClusterMasterHealthEntry represents a site of an indexer cluster with its metadata
+type ClusterMasterHealthEntry struct {
+	Name    string                     `json:"name"`
+	Content ClusterMasterHealthContent `json:"content"`
+}
+
+// ClusterMasterHealthContent represents detailed information about a site
+type ClusterMasterHealthContent struct {
+	AllDataIsSearchable      string `json:"all_data_is_searchable"`
+	AllPeersAreUp            string `json:"all_peers_are_up"`
+	Multisite                string `json:"multisite"`
+	NoFixupTasksInProgress   string `json:"no_fixup_tasks_in_progress"`
+	ReplicationFactorMet     string `json:"replication_factor_met"`
+	SearchFactorMet          string `json:"search_factor_met"`
+	SiteReplicationFactorMet string `json:"site_replication_factor_met"`
+	SiteSearchFactorMet      string `json:"site_search_factor_met"`
+}
```

Comment:
Aren't these be best represented with `bool` than `string`

Reply:
@sgontla the return from Splunk API is string.